prow/cmd/phaino/local.go

```diff
@@ -55,11 +55,16 @@ func scanln(ctx context.Context) (string, error) {
 }
 
 func readMount(ctx context.Context, mount coreapi.VolumeMount) (string, error) {
-	fmt.Fprintf(os.Stderr, "local %s path (%q mount): ", mount.MountPath, mount.Name)
+	defaultMount := "mounts/" + mount.Name
```

Comment:
Hmm, this seems potentially dangerous? Is it common to have a /go/src/k8s.io/test-infra/mounts directory?

How about if we at least check for the existence of this directory before recommending it as the default? (like we do for trying to find the repo)